Draw/Elements/UI/NavigationItemAPI.cs

```diff
@@ -102,5 +102,15 @@ public List<PageTagAPI> tags
             get;
             set;
         }
+
+         /// <summary>
```

Comment:
This stuff shouldn't be on this branch - I'm guessing you accidentally based it on the branch for your navigation changes?

Reply:
I have no idea how I managed this! Will create a new branch and PR.